app/src/main/java/io/intrepid/contest/screens/contestcreation/describecontest/DescribeContestPresenter.java

```diff
@@ -14,15 +13,30 @@
     }
 
     @Override
+    public void onViewCreated() {
+        super.onViewCreated();
+        view.disableNext();
+    }
+
+    @Override
     public void onNextClicked(String description) {
```

Comment:
Spoke with them. this should be removed. The conditional check will never be used.